src/lib/openapi/spec/resource-limits-schema.ts

```diff
@@ -82,6 +83,12 @@ export const resourceLimitsSchema = {
             example: 50,
             description: 'The maximum number of environments allowed.',
         },
+        projects: {
+            type: 'integer',
+            minimum: 1,
```

Comment:
this is also enforced in the service

Reply:
I'm sure you know this, but the limit here really has no bearing on what goes on in Unleash, right? This is just a promise we make to our end users. And in fact, in this case, if you set it to 0, you'll get 0 out. Unleash will say the schema doesn't allow the value, but it won't correct it for you.

Unless I'm missing something?

Reply:
I wanted to communicate that you'll never get 0 in the GET request for UI config with the resource limits. The parsing (`Math.max(1, parseEnvVarNumber(process.env.UNLEASH_PROJECTS_LIMIT, 500))`) we're doing should provide this guarantee